package.json

```diff
@@ -28,8 +29,21 @@
     "prop-types": "^15.5.10",
     "react-native-permissions": "^1.1.1"
   },
+  "devDependencies": {
+    "babel-eslint": "^8.2.3",
+    "eslint": "^4.19.1",
+    "husky": "^0.14.3",
+    "lint-staged": "^7.1.0",
+    "prettier": "^1.12.1"
+  },
   "peerDependencies": {
     "react-native-camera": "^1.0.2"
   },
-  "homepage": "https://github.com/moaazsidat/react-native-qrcode-scanner#readme"
+  "homepage": "https://github.com/moaazsidat/react-native-qrcode-scanner#readme",
+  "lint-staged": {
+    "*.js": [
+      "prettier --single-quote --trailing-comma es5 --tab-width 4 --write",
```

Comment:
can we change tab width to 2? otherwise looks good, thanks!

Reply:
Done!